### DISCUSSION TITLE
Gyrodrivebase for EV3

### DISCUSSION THREAD

**Discussion by RoboticBuilders** [pybricks/support#1566] [General]
Hello,
We are using pybricks with EV3 for a competition and missing the awesome Gyrodrivebase functionality from pybricks for Spike Prime. Is there anyway to specify the port for the gyro and then use the GyroDrivebase in EV3? Also is there no drivebase.curve functionality in EV3?
Thanks so much for your help!

**Comment by laurensvalk** [comment]
Unfortunately the external gyro doesn't provide enough resolution and control to get the same kind of accuracy as we get on Spike, so this hasn't been implemented for EV3.

Pybricks for EV3 is still at version 2.0 while the curve functionality was introduced in version 3.3.

We'd like to update the EV3 at some point but there isn't a definitive plan or timeline for it at the moment.
